index.js

```diff
@@ -1,14 +1,15 @@
 "use strict";
 
+var munch = require("./lib/munch");
+var strip = require("./lib/strip");
+var fileFinder = require("./lib/fileFinder");
+var docxChooser = require("./lib/docxChooser");
 var woolly = require("./lib/woolly-mammoth");
 var normalize = require("./lib/normalize");
-var markout = require("./lib/markout");
 var paginate = require("./lib/paginate");
+var markout = require("./lib/markout");
+var statsTracker = require("./lib/statsTracker");
```

Comment:
Note, the `statsTracker` is the only new addition.  The remainder of the changes was a reorganization.